scripts/build_utils.sh

```diff
@@ -1072,3 +1072,12 @@ get_nr_build_jobs() {
     fi
     echo $n_build_jobs
 }
+
+maybe_reset_ci_container() {
+    # ceph-container includes ceph-mgr-cephadm by default, but nautilus does
+    # not ship this package
```

Comment:
which sounds strange. isn't it the job of ceph-container to decide which ceph version ships which packages? anyway. this PR lgtm.

Reply:
> which sounds strange. isn't it the job of ceph-container to decide which ceph version ships which packages? 

right. but nautilus does not have ceph-mgr-cephadm at all. ceph-container cannot ship the package which does not even exist in upstream.